# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 5194f8d4d589
# Full Hash: 5194f8d4d5890aafe62c42f9b1fa75d6a9a64bf5
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P12. Allow ffvpx and the AppleDecoderModule in the RDD. r=haik,jolin
#   
#   Ultimately, we should be able to remove everything that got added to the RDD sandbox from the content's one.
#   
#   Fly-by fix; allow checking if AVX512 is supported in content sandbox.
# ==============================================================================

diff -r 84b82a60dc74 -r 5194f8d4d589 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Oct 20 23:27:36 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Oct 20 23:27:52 2020 +0000
@@ -7081,6 +7081,8 @@
   type: RelaxedAtomicBool
 #if defined(XP_WIN) && !defined(_ARM64_)
   value: true
+#elif defined(XP_MACOSX)
+  value: true
 #else
   value: false
 #endif
@@ -7098,6 +7100,13 @@
   mirror: always
 #endif
 
+#ifdef MOZ_APPLEMEDIA
+- name: media.rdd-applemedia.enabled
+  type: RelaxedAtomicBool
+  value: true
+  mirror: always
+#endif
+
 - name: media.rdd-vorbis.enabled
   type: RelaxedAtomicBool
 #if defined(XP_LINUX) && !defined(ANDROID)